docs/modules/dev/pages/manual_build.adoc

```diff
@@ -148,16 +152,43 @@ apt-get install -y --no-install-recommends \
 .Compile
 [source,bash]
 ....
-cmake -Bbuild -DCMAKE_C_FLAGS=$CFLAGS -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DCMAKE_CXX_EXTENSIONS=OFF -G Ninja
+cmake -Bbuild -DCMAKE_C_FLAGS="$CFLAGS" -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DCMAKE_CXX_EXTENSIONS=OFF -G Ninja
 ninja -C build
 ....
 
-If compilation completes correctly, you can finally start Wolf
+If compilation completes correctly, you can finally start Wolf. The built binary can be found at `build/src/moonlight-server/wolf`
+Since Wolf is configured via a swoth of environment variables, it may be a good idea to lanch it via shell script.
+
+.runwolf.sh
+[source,bash]
+....
+#!/bin/bash
+
+: ${WOLF_CFG_FOLDER="config"}
+
+XDG_RUNTIME_DIR="${XDG_RUNTIME_DIR=/tmp/sockets}" \
+WOLF_LOG_LEVEL="${WOLF_LOG_LEVEL=INFO}" \
+WOLF_CFG_FILE="${WOLF_CFG_FILE=$WOLF_CFG_FOLDER/config.toml}" \
+WOLF_PRIVATE_KEY_FILE="${WOLF_PRIVATE_KEY_FILE=$WOLF_CFG_FOLDER/key.pem}" \
+WOLF_PRIVATE_CERT_FILE="${WOLF_PRIVATE_CERT_FILE=$WOLF_CFG_FOLDER/cert.pem}" \
+WOLF_PULSE_IMAGE="${WOLF_PULSE_IMAGE=ghcr.io/games-on-whales/pulseaudio:master}" \
+WOLF_RENDER_NODE="${WOLF_RENDER_NODE=/dev/dri/renderD128}" \
+WOLF_STOP_CONTAINER_ON_EXIT="${WOLF_STOP_CONTAINER_ON_EXIT=TRUE}" \
+WOLF_DOCKER_SOCKET="${WOLF_DOCKER_SOCKET=/var/run/docker.sock}" \
+RUST_BACKTRACE="${RUST_BACKTRACE=full}" \
+RUST_LOG="${RUST_LOG=WARN}" \
+HOST_APPS_STATE_FOLDER="${HOST_APPS_STATE_FOLDER=$WOLF_CFG_FOLDER}" \
+GST_DEBUG="${GST_DEBUG=2}" \
+PUID="${PUID=0}" \
+PGID="${PGID=0}" \
```

Comment:
These two aren't used by Wolf, they are just used by the Docker container to setup the running user